back/app/services/concerns/json_forms_ideas_overrides.rb

```diff
@@ -44,12 +44,47 @@ def custom_form_to_ui_schema fields, locale='en'
           type: 'Category',
           options: { id: 'extra' },
           label: I18n.t('custom_forms.categories.extra.title', locale: locale),
-          elements: fields.reject(&:built_in?).map{|f| yield f}
+          elements: fields.reject(&:built_in?).map{|f| yield f, previousScope = '#/properties/custom_field_values/properties/'}
```

Comment:
Since these elements are nested in the jsonschema, I pass in this previous scope to get the resulting scope and adapated the uischema methods to concatenate the scope they compute to this one.